test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -363,6 +369,32 @@ def test_successful_retrieve_subscription
     assert response.test?
   end
 
+  def test_3ds_purchase_request
+    assert response = @gateway.purchase(1202, @three_ds_enrolled_card, @options.merge(payer_auth_enroll_service: true))
```

Comment:
I wonder if we have other gateways that send a similar, "check if the card is 3DS enabled" field and we've defaulted in Active Merchant? Not sure, but could be worth a check to see if there's a more or less default field we use for this.

Reply:
Yeah it's kinda difficult to determine, which probably means there's no standard. I'm fine with keeping the option field verbatim to the actual request field sent until there's a more concerted effort to standardize.